qir/qat/Version/Version.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "qir/qat/Version/Version.hpp"
+
+#include <cstdint>
+#include <iostream>
+#include <string>
+
+namespace microsoft::quantum
+{
+
+void displayCliHeader(bool extended_info)
+{
+    std::cout << "QAT v. " << version::FULL << std::endl;
+    std::cout << "Copyright 2021-2022 (c) Microsoft Corporation." << std::endl;
```

Comment:
Why does this copyright differ from the one in line 1?

Reply:
Changed.